web-registry/src/graphql/GetWalletByAddress.graphql

```diff
@@ -2,7 +2,10 @@ query walletByAddr($addr: String!) {
   walletByAddr(addr: $addr) {
     id
     addr
-    projectsByWalletId {
+    partyByWalletId {
```

Comment:
I created a dedicated request for that in case you want only need that part of the response: https://github.com/regen-network/regen-web/pull/1821/files#diff-9fe5d10822484959e7f786163a30b5269be2cad5d0680cfb01faba23a8e7cac7